sciences-po-ecole-doctorale-note-french.csl

```diff
@@ -0,0 +1,423 @@
+      <term name="ordinal-01">ère</term>
+      <term name="ordinal-02">e</term>
+      <term name="ordinal-03">e</term>
+      <term name="ordinal-04">e</term>
+      <term name="cited">Op.&#160;cit.</term>
+      <term name="page" form="short">p.</term>
+      <term name="editor" form="short">
+        <single>ed.</single>
+        <multiple>eds.</multiple>
+      </term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <names variable="author">
+      <name and="text" name-as-sort-order="first" sort-separator=", ">
+        <name-part name="family" font-variant="small-caps"/>
+      </name>
+      <label form="short" prefix="&#160;(" suffix=")"/>
+      <substitute>
+        <names variable="editor"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="editor">
+    <names variable="editor">
+      <name form="long" and="text" delimiter-precedes-last="never" sort-separator=", " font-style="normal">
+        <name-part name="family" font-variant="small-caps"/>
+      </name>
+      <label form="short" prefix="&#160;(" suffix=")"/>
+    </names>
+  </macro>
+  <macro name="translator">
+    <names variable="translator">
+      <label form="verb" suffix=" "/>
+      <name form="long" and="text" delimiter-precedes-last="never" sort-separator=" ">
+        <name-part name="family" font-variant="small-caps"/>
+      </name>
+    </names>
+  </macro>
+  <macro name="archive">
+    <choose>
+      <if variable="archive archive_location" match="any">
+        <group delimiter=". ">
+          <text variable="archive_location" text-case="capitalize-first"/>
+          <text variable="archive"/>
+          <text variable="archive-place"/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="scale">
+    <choose>
+      <if type="map" match="any">
+        <text variable="scale" prefix="Echelle "/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="medium">
+    <choose>
+      <if type="graphic motion_picture" match="any">
+        <text variable="medium"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="number">
+    <choose>
+      <if type="patent" match="any">
+        <text variable="number" prefix="Brevet "/>
+      </if>
+      <else-if type="report" match="any">
+        <text variable="number" prefix="Rapport "/>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="dimensions">
+    <choose>
+      <if type="motion_picture graphic broadcast" match="any">
+        <text variable="dimensions"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="originally-published">
+    <date variable="original-date" form="text" date-parts="year" prefix="[" suffix="] "/>
+  </macro>
+  <macro name="version">
+    <choose>
+      <if type="book" match="any">
+        <text variable="version" prefix="Version "/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if variable="URL" match="any">
+        <text variable="URL" prefix="En ligne : " suffix=" "/>
```

Comment:
Shouldn't this and the two other instances of " : " be `&160;: ` instead? Without the non-breaking space, you can have a line starting with :, which surely can't be desirable.

Reply:
ok, thank you for warning about that !